src/notifications/lipsticknotification.h

```diff
@@ -46,6 +46,7 @@ class LIPSTICK_EXPORT LipstickNotification : public QObject
     Q_PROPERTY(int priority READ priority NOTIFY priorityChanged)
     Q_PROPERTY(QString category READ category NOTIFY categoryChanged)
     Q_PROPERTY(bool userRemovable READ isUserRemovable NOTIFY userRemovableChanged)
+    Q_PROPERTY(QVariantList remoteActions READ remoteActions)
```

Comment:
There's setActions() as public method, so technically the property can change too. Though seems actions property and some others are missing signal already.